client/client-multi/client-c/test/kaa_tcp_channel/test_kaa_tcp_channel_bootstrap.c

```diff
@@ -730,16 +730,21 @@ kaa_error_t kaa_platform_protocol_alloc_serialize_client_sync(kaa_platform_proto
 
 ext_tcp_socket_state_t ext_tcp_utils_tcp_socket_check(kaa_fd_t fd, const kaa_sockaddr_t *destination, kaa_socklen_t destination_size)
 {
+    /* Only to avoid compiler's warning */
```

Comment:
You can avoid such comments in the future. Such usages of cast to void are common.